src/main/java/UseCase/playSpaceInteractor.java

```diff
@@ -1,16 +1,34 @@
 package UseCase;
 
+import Entities.Song;
 import InputData.spaceInputData;
-import OutputBoundary.spacePlayedOutputBoundary;
 import OutputData.songOutputData;
+import java.io.File;
 
+/**
+ * use case for playing a space
+ */
 public class playSpaceInteractor {
 
     private OutputBoundary.spacePlayedOutputBoundary spacePlayedOutputBoundary;
```

Comment:
You would need to have a constructor to initialise this instance variable I think. This would allow us to pass in a Presenter object so that when you call the spacePlayed method later on, you are actually calling it from the Presenter class.